zerver/models.py

```diff
@@ -684,6 +684,8 @@ class Realm(models.Model):
 
     default_code_block_language: Optional[str] = models.TextField(null=True, default=None)
 
+    enable_read_receipts: bool = models.BooleanField(default=False)
```

Comment:
Let's add a comment for this one -- "Whether read receipts are enabled in the organization. If disabled, they will not be available regardless of users' personal settings."